x-pack/legacy/plugins/apm/cypress/screenshots/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
do we need two .gitignore files? maybe easier to have just one in `x-pack/legacy/plugins/apm/cypress`?

Reply:
True, I'll combine them.